frontend/src/lib/components/ProjectList.svelte

```diff
@@ -73,4 +104,9 @@
       @apply opacity-100;
     }
   }
+
+  .draft.card {
+    pointer-events: none;
+    box-shadow: none;
```

Comment:
See my other comment about removing `shadow-base-300`
```suggestion
```

Reply:
Hey @myieye 👋  Removing `shadow-base-300` does not seem to completely disable the box shadow. To fully remove the box shadow, we need to keep `box-shadow: none;` in the `.draft.card` class to override(?) any effects. Please let me know if there are any better ways to make this work!

Reply:
As mentioned above, I think this is necessary in order to override the specificity of `.card`.